base/src/com/thoughtworks/go/util/FileUtil.java

```diff
@@ -64,21 +76,18 @@ public static String makepath(String... paths) {
     }
 
     public static String readToEnd(File file) throws IOException {
-        return readToEnd(new FileInputStream(file));
+        FileInputStream input = new FileInputStream(file);
+        return readToEnd(input);
     }
 
     public static String readToEnd(InputStream input) throws IOException {
-        try {
-            @SuppressWarnings("unchecked") List<String> list = IOUtils.readLines(input);
-            StringBuilder builder = new StringBuilder();
-            for (String line : list) {
-                builder.append(line);
-                builder.append(lineSeparator());
-            }
-            return builder.toString().trim();
-        } finally {
-            IOUtils.closeQuietly(input);
+        @SuppressWarnings("unchecked") List<String> list = IOUtils.readLines(input);
+        StringBuilder builder = new StringBuilder();
+        for (String line : list) {
+            builder.append(line);
+            builder.append(lineSeparator());
         }
+        return builder.toString().trim();
```

Comment:
`IOUtils.closeQuietly` looks harmless, but so did the original PR :laughing: . The intent was to leave it in the form it was earlier. I can still add it if you think its necessary?
Weren't you off to sleep?